R/timeNumeric.R

```diff
@@ -0,0 +1,49 @@
+#' Time field provided in character as "%H:%M:%S" or in Period class ("H M S")
+#' can be converted to numeric values depending on the units.
+#'
+#' @param time_field : vector containing the time points
+#' @param units : The unit of time points selected in `time_field`.
+#' It can be hours, minutes or seconds.(default: \code{unit = "mins"})
+#'
+#' @examples
+#' data(hitchip1006)
+#' hitchipTime <- timeToDate(time_field = colData(hitchip1006)[,"time"],
```

Comment:
hitchip time is in days, fix here and in other places

Reply:
I can now see that this format does not support days. Is this something we cannot overcome easily?